src/components/TodoComponents/TodoForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const TodoForm = props => {
+
+  return (
+
+    <form>
+      <input onChange={props.handleTodoChange} type="text" name="todo" value={props.value} placeholder="...todo"/>
+
+      <button onClick={props.handleAddTodo}>Add Todo</button>
+
+      <button onClick={props.handleClearTodos}>Clear Completed</button>
+    </form>
```

Comment:
you'll need to define this method and also remember to add the prventDefault method to stop the form from reloading when you click